examples/geospatial-python-urban-analysis-with-postgis/src/visualization/optimization_map.py

```
@@ -0,0 +1,35 @@
+import matplotlib.pyplot as plt
+import contextily as ctx
+import geopandas as gpd
```

Ruff (0.8.2)

`geopandas` imported but unused

Remove unused import: `geopandas`

(F401)